lib/lambdalocal.js

```diff
@@ -6,81 +6,76 @@
  * https://docs.aws.amazon.com/en_us/lambda/latest/dg/nodejs-prog-model-context.html
  */
 
-const mute = require('mute'),
-  utils = require('./utils.js'),
-  _context = require('./context.js');
+var logger = require('winston');
 
+const mute = require('mute'),
+    utils = require('./utils.js'),
+    _context = require('./context.js');
+	
+var _setLogger = function(_logger){
+	logger = _logger;
```

Comment:
It is a little scary that this function takes any object and sets it as its logger. It would be better if `_setLogger` checks if the logger is a `winston` object. 
Of course developers will receive an error when the `logger` fails to call the `log()` function, so it isn't critical. I just think it'd be a little cleaner if `_setLogger` will validate the object. 

Therefore this is not a must-do.